sylt-parser/src/statement.rs

```diff
@@ -457,7 +456,13 @@ pub fn statement<'t>(ctx: Context<'t>) -> ParseResult<'t, Statement> {
         }
     };
 
-    let ctx = ctx.skip_if(T::Newline);
+    // Newline, RightBrace and Else can end a statment.
+    // We don't give too specific errors when that is not the case.
```

Comment:
```suggestion
    // Errors only report on the newline - since it's usually the solution.
```
Maybe? I dislike the double negation in the comment - makes it hard to follow.

Reply:
What's "the solution"?

Reply:
```suggestion
    // If a statement does not end, we only report it as a missing newline.
```
Perhaps?